schemas/mod.graphql

```diff
@@ -13,6 +13,7 @@ enum ModFields {
     popularity
     last_version_date
     search
+    tags
```

Comment:
This is only used for fields that can be ordered by. A list of tags cannot be ordered by